contracts/Vault.vy

```diff
@@ -229,7 +229,11 @@ managementFee: public(uint256)
 # Governance Fee for performance of Vault (given to `rewards`)
 performanceFee: public(uint256)
 MAX_BPS: constant(uint256) = 10_000  # 100%, or 10k basis points
-SECS_PER_YEAR: constant(uint256) = 31_557_600  # 365.25 days
+# NOTE: A four-century period will be missing 3 of its 100 Julian leap years, leaving 97. 
+#       So the average year has 365 + 97/400 = 365.2425 days 
+#       ERROR(Julian): -0.0078
+#       ERROR(Gregorian): -0.0003
+SECS_PER_YEAR: constant(uint256) = 31_556_926  # 365.24 days
```

Comment:
```suggestion
SECS_PER_YEAR: constant(uint256) = 31_556_952  # 365.2425 days
```

Reply:
previous constant was only to two decimal places hence why I only placed the constant value to that value.